docs/user_guide/1_setting_up_the_hand.md

```diff
@@ -47,15 +49,14 @@ There are two ways to connect the EtherCAT and power cables to the hand.
 If your hand already has cables fitted, then you can simply connect the EtherCAT and power connectors immediately.
 ![Connecting cables](../img/connecting_cables_external.png)
 
-**EtherCAT**: Connect the Ethernet cable to the hand's Ethernet socket, and connect the other end to the PC's second
-Ethernet port. **If you have a Bi-manual system, connect the Left and Right hands correctly to the labelled ports.**
-You have been supplied with a medium length Ethernet lead, but if you require a longer or shorter one, you can simply use a standard commercial Ethernet Cat 5 cable, available from most computer parts suppliers.
+**EtherCAT**: Connect the Ethernet cable to the hand's Ethernet socket, and connect the other end to the USB->Ethernet adapter with a label `HAND`. Then, connect the USB end of the adapter to any of the USB ports in the NUC. Next, connect USB->Ethernet adapter with a label `CONTROL MACHINE` to another USB port on the NUC and adapter with a label `CLIENT MACHINE` to any of the ports in your client PC (provided by Shadow or a custom one). Finally, connect the two adaptors together with an Ethernet cable.
```

Comment:
My delivery instructions have "NUC-CONTROL" (the hostname of the NUC) rather than "CONTROL MACHINE". Also the laptop is called "SERVER" (which matches the laptop's hostname), not "CLIENT MACHINE". I suggest we keep calling the laptop the "SERVER" (this is where the ROS_MASTER_URI is, not CLIENT MACHINE. I also suggest we keep the NUC labelled as "NUC-CONTROL" because that matches the hostname.